pages/components/feed.js

```diff
@@ -1,12 +1,18 @@
 import Cards from "./cards";
 import ActiveMusic from "./activeMusic";
 import Navbar from "./navBar";
+import styled from "styled-components";
+
 export default function Feed() {
   return (
-    <div>
+    <FeedSection>
       <ActiveMusic />
       <Cards />
       <Navbar />
-    </div>
+    </FeedSection>
   );
 }
+
+const FeedSection = styled.div`
```

Comment:
Why not use a section element if its already named after it?:D

Reply:
what do you mean. Did not understand  :O